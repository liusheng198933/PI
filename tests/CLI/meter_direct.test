#!/bin/sh
./run_one_test.py $srcdir/testdata meter_direct stats.json
