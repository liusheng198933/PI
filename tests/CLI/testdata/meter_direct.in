meter_read_spec ExactOne_meter 0
meter_set ExactOne_meter NEXT_ENTRY 1:1 2:2
table_add ExactOne 0 => actionA 0
meter_read_spec ExactOne_meter 0
meter_set ExactOne_meter 0 1:3 2:12
meter_read_spec ExactOne_meter 0
meter_set ExactOne_meter 0 400:300 1000:300
meter_read_spec ExactOne_meter 0
