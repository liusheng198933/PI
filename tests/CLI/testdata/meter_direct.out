????
Device assigned successfully.
Selecting device.
????
Error when trying to read meter spec
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 1
	Committed burst size: 1
	Peak information rate: 2
	Peak burst size: 2
Direct resource spec was stored.
????
Entry was successfully added with handle 0.
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 1
	Committed burst size: 1
	Peak information rate: 2
	Peak burst size: 2
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 1
	Committed burst size: 3
	Peak information rate: 2
	Peak burst size: 12
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 1
	Committed burst size: 3
	Peak information rate: 2
	Peak burst size: 12
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 400
	Committed burst size: 300
	Peak information rate: 1000
	Peak burst size: 300
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 400
	Committed burst size: 300
	Peak information rate: 1000
	Peak burst size: 300
????
